src/components/generic/DisplayTime.tsx

```diff
@@ -0,0 +1,18 @@
+interface DisplayTimeProps {
+    minutes?: number;
+    seconds?: number;
+    hundredths?: number;
+}
+
+const DisplayTime: React.FC<DisplayTimeProps> = ({ minutes = 0, seconds = 0, hundredths = 0 }) => {
+    // Ensure two digits for minutes, seconds, annd miliseconds
+    const formatTime = (value: number, digits = 2) => value.toString().padStart(digits, '0');
```

Comment:
this should live outside the component if it doesn't depend on anything in it, otherwise the function gets redefined on every re-render